unit_test/solver/network/test_network.py

```diff
@@ -461,13 +463,13 @@ def test_set_result_asset(self) -> None:
         self.network.set_result_asset(solution=solution)  # act
 
         # assert
-        self.assertEqual(self.asset.prev_sol, solution[:6])
-        self.assertEqual(asset2.prev_sol, solution[6:])
+        self.assertEqual(self.asset.prev_sol, solution[:6].tolist())
```

Comment:
can be done with numpy as well, see https://numpy.org/doc/stable/reference/routines.testing.html

Reply:
let's discuss

Reply:
This is possible, but better to do when we also change the prev_sol list to np.array. I will make issue out of this